includes/ApplHandlingByStr.php

```diff
@@ -74,6 +74,10 @@ function showFullStatus( $stat_str )
 		{
 			echo "<td><b>FORWARDED</b> from ".$str[0]."</td>";
 		}
+		else if( $str[1] == 'G' )
+		{
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->